scripts/exp_userspace.py

```
@@ -0,0 +1,162 @@
+        )
+        rec["sshd_uptime"] = mk.get("sshd-invoked")
+    finally:
+        if vm is not None:
+            try: vm.stop(); vm.delete()
+            except Exception: pass
+    return rec
+
+
+def run(label, kernel, rootfs, tag):
+    print(f"\n== {label}: warm-up ==", flush=True)
+    try: one(kernel, rootfs, tag)
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)